src/client/NuGet.CommandLine.StagingExtensions/StageManagementResource.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Client.Staging;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.v3;
+
+namespace NuGet.CommandLine.StagingExtensions
+{
+    public class StageManagementResource : Protocol.Core.Types.INuGetResource
+    {
+        private const string ApiKeyHeader = "X-NuGet-ApiKey";
+        private const string StagePath = "api/stage";
+        private const string MediaType = "application/json";
+
+        private HttpSource _httpSource;
+        private string _stageServiceUri;
+
+        public StageManagementResource(string stageServiceUri, HttpSource httpSource)
+        {
+            if (string.IsNullOrEmpty(stageServiceUri))

```

Comment:
Removed those null checks. The values are null checked in StageManagementResourceProvider, and can't be null here. Push Command follows the same pattern.

---
In reply to: [66826605](https://github.com/NuGet/NuGet.Staging/pull/62#discussion_r66826605) [](ancestors = 66826605)